docs/packages/pkg/qhull.rst

```diff
@@ -0,0 +1,23 @@
+.. spelling::
+
+    qhull
+
+.. index:: unsorted ; qhull
+
+.. _pkg.qhull:
+
+qhull
+=====
+
+-  `Official <https://github.com/qhull/qhull>`__
+-  `Hunterized <https://github.com/hunter-packages/qhull>`__
+-  `Example <https://github.com/hunter-packages/qhull/blob/hunter-7.2.0/CMakeLists.txt>`__
```

Comment:
No, it should be `https://github.com/ruslo/hunter/blob/master/examples/qhull/CMakeLists.txt`, see template file:
* https://raw.githubusercontent.com/ruslo/hunter/v0.23.36/docs/packages/pkg/foo.rst

And documentation:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation